protobuf-solidity/src/protoc/plugin/gen_sol.py

```diff
@@ -253,6 +255,27 @@ def apply_options(params_string):
   if "solc_version" in params:
     util.set_solc_version(params["solc_version"])
 
+def change_runtime_file_names(name: str):
+  if name.endswith(".sol"):
```

Comment:
can we raise an exception if the condition is `else`?

Reply:
Yes, I'll fix it immediately.

Reply:
done